src/main/java/com/minecolonies/core/network/messages/server/colony/ColonyDeleteOwnMessage.java

```diff
@@ -53,18 +51,6 @@ public void onExecute(final NetworkEvent.Context ctxIn, final boolean isLogicalS
         {
             IColonyManager.getInstance().deleteColonyByDimension(colony.getID(), false, colony.getDimension());
             MessageUtils.format(MESSAGE_INFO_COLONY_DESTROY_SUCCESS).sendTo(player);
-
-            if (isLogicalServer)
-            {
-                try
-                {
-                    MinecraftForge.EVENT_BUS.post(new ColonyDeletedEvent(colony));
```

Comment:
?

Reply:
The event was moved to the ColonyManager deleteColony method. I noticed this event was only fired upon deletion of your colony through the townhall GUI, not from the delete command.

So I moved that event call in there in favor of here.

Reply:
kk